perfkitbenchmarker/linux_benchmarks/mnist_benchmark.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Run MNIST benchmarks."""
+
+import re
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+
+FLAGS = flags.FLAGS
+
+BENCHMARK_NAME = 'mnist'
+BENCHMARK_CONFIG = """
+mnist:
+  description: Runs MNIST Benchmark.
+  vm_groups:
+    default:
+      vm_spec:
+        GCP:
+          image: debian-9-tf-v20171009
+          image_project: ml-images
+          machine_type: n1-standard-8
+          zone: us-central1-c
+  cloud_tpu:
```

Comment:
I notice that you don't specify tpu_accelerator_type here, and there is no default in the flags or the spec. What default is gcloud using? Maybe we should add a default value in PKB to be clear.

Reply:
The default tpu_accelerator_type is tpu-v2. If not specified in "gcloud compute tpus create", the default is tpu-v2. We can give a value in PKB. If we don't give, it will use the default tpu_accelerator_type value from this command "gcloud compute tpus create", which is tpu-v2

Reply:
Ok, let's just be explicit and set the default flag value in PKB to tpu-v2.